easybuild/easyconfigs/m/magma/make.inc

```diff
@@ -0,0 +1,49 @@
+#//////////////////////////////////////////////////////////////////////////////
```

Comment:
If you run EB with the `-l` flag (or `--debug`), you will seen that EB sets a whole bunch of env vars for compilers, library paths, etc. Please try to use those as much as possible: set the variables in this file to those environment variables.